lib/brains/forms.py

```diff
@@ -1,5 +1,40 @@
 from django import forms
+from django.core.exceptions import ObjectDoesNotExist
+
+from lib.buyers.models import Buyer
 
 
 class BuyerForm(forms.Form):
     uuid = forms.CharField(max_length=255)
+
+
+class PaymentMethodForm(forms.Form):
+    buyer_uuid = forms.CharField(max_length=255)
+    nonce = forms.CharField(max_length=255)
+
+    def clean_buyer_uuid(self):
+        data = self.cleaned_data
+
+        try:
+            self.buyer = Buyer.objects.get(uuid=data['buyer_uuid'])
+        except ObjectDoesNotExist:
+            raise forms.ValidationError('Buyer does not exist.')
```

Comment:
mmm, we need to figure out localization soon. We started doing it by raising mappable constant strings like https://github.com/mozilla/webpay/blob/master/webpay/pay/forms.py#L57 . I think that's the right way to approach it but we could punt this until later I suppose.